### PR TITLE
BONK 10 Support

### PR DESCRIPTION
**Pull request by DonDuala**

### CODE REVIEW

**Review by brittcyr — APPROVED** [review]

File: src/config.ts [hunk 1]
```diff
@@ -118,6 +118,7 @@ export const BVE_VOL_MAP = new Map<SYMBOL, number>([
 ]); // BVE vol keep alligned with app & contract vol
 
 export const SO_STATES: [string, PublicKey][] = [
+  ['GSOBONK_LOYALTY_10', new PublicKey('47JR8VXLc32X74ZFHcSFwkdZLWLdrU1f3WxDtgQUBhwj')],
```
Comment (brittcyr) [inline src/config.ts:121]: Not used, but this is supposed to be the state address

Comment (DonDuala) [reply src/config.ts:121]: Oh only name matters?

Comment (brittcyr) [reply src/config.ts:121]: for now. only because we assume the baseMint is BONK which lets us compute the state address.

Comment (DonDuala) [reply src/config.ts:121]: Will handle with the updates for MNGO then

**Pull request merged**